app/services/gpo_reminder_sender.rb

```diff
@@ -1,9 +1,13 @@
 class GpoReminderSender
   def send_emails(for_letters_sent_before)
+    letter_eligible_range =
+      IdentityConfig.store.usps_confirmation_max_days.days.ago..for_letters_sent_before
+
     profiles_due_for_reminder = Profile.joins(:gpo_confirmation_codes).
       where(
-        gpo_verification_pending_at: ..for_letters_sent_before,
+        gpo_verification_pending_at: letter_eligible_range,
```

Comment:
I think this _should_ work. I can't figure out a way that we would send a letter to user who is not GPO eligible. If a case does exist I'm pretty confident it is a rare corner case.